src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example1.java

```diff
@@ -18,23 +19,24 @@ public static void fun() {
   }
 }
 
-class Foo { // OK
+class Foo1 { // OK
```

Comment:
please remove all '"// OK", no violation means ok, we missed it in previos updates